prow/cmd/jenkins-operator/main.go

```diff
@@ -111,6 +117,13 @@ func main() {
 	go serveLogs(jc)
 
 	for range time.Tick(30 * time.Second) {
+		select {
```

Comment:
Can you combine this and the `for range time.Tick(...)` into one `select`?

Reply:
ack